public/embed/index.html

```diff
@@ -39,10 +39,10 @@
     <link rel="stylesheet" href="../css/embed.css">
     <link rel="stylesheet" href="../css/styles.css" />
     <link rel="stylesheet" href="../css/map.css" />
-    <script src="../languages/He.js"></script>
-    <script src="../languages/En.js"></script>
-    <script src="../languages/Ar.js"></script>
     <script src="../languages/i18n.js"></script>
+    <script src="../languages/he.js"></script>
```

Comment:
is it after i18n on purpose?

Reply:
Yup. Each language "registers itself" in a map variable that `i18n.js` declares.